homeassistant/components/deconz/cover.py

```diff
@@ -85,52 +99,52 @@ def __init__(self, device, gateway):
         self._attr_device_class = DEVICE_CLASS.get(self._device.type)
 
     @property
-    def current_cover_position(self):
+    def current_cover_position(self) -> int:
         """Return the current position of the cover."""
-        return 100 - self._device.lift
+        return 100 - self._device.lift  # type: ignore[no-any-return]
 
     @property
-    def is_closed(self):
+    def is_closed(self) -> bool:
         """Return if the cover is closed."""
         return not self._device.is_open
 
-    async def async_set_cover_position(self, **kwargs):
+    async def async_set_cover_position(self, **kwargs: int) -> None:
```

Comment:
`**kwargs` are always typed with `Any`. It's not an integer. The position parameter is an integer but they are not the same.

Reply:
Thanks! I will fix that

Reply:
Fixed here https://github.com/home-assistant/core/pull/59997